x11-wm/selectwm/selectwm-0.4.1.ebuild

```diff
@@ -1,27 +1,30 @@
-# Copyright 1999-2015 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
-inherit autotools eutils
+EAPI=6
+inherit autotools
 
 DESCRIPTION="window manager selector tool"
-HOMEPAGE="http://ordiluc.net/selectwm"
-SRC_URI="http://ordiluc.net/selectwm/${P}.tar.bz2"
+HOMEPAGE="https://ordiluc.net/selectwm"
+SRC_URI="https://ordiluc.net/selectwm/${P}.tar.bz2"
 
 LICENSE="GPL-2"
 SLOT="2"
 KEYWORDS="amd64 ppc sparc x86 ~x86-fbsd"
 IUSE="nls"
 
+DOCS=( AUTHORS README sample.xinitrc )
+
 RDEPEND="x11-libs/gtk+:2
 	dev-libs/glib:2"
 DEPEND="${RDEPEND}
 	virtual/pkgconfig"
 
 src_prepare() {
-	epatch "${FILESDIR}"/${P}-enable-deprecated-gtk.patch \
+	eapply "${FILESDIR}"/${P}-enable-deprecated-gtk.patch \
```

Comment:
Can you use PATCHES=() ?

Reply:
Hi, i've updated the PR to use the PATCHES Variable